### ISSUE TITLE
关于数据方面的问题

### ISSUE THREAD

**Issue by 70-yb** [geekdanielz/DeepVANet#1]
你好，我在数据处理的步骤方面有些疑问。如果是按测试者来分别做实验，在脑电上一个测试者有40trial*60s=2400个数据，那么面部图片数据必须要和脑电对应吗？
以及以下代码循环没太理解：
            for n in range(1, 6):
            img = Image.open(io.BytesIO(self.face_zip.read(prex + f'_{(segment - 1) * 5 + n}.png')))
            frame_array = transform(img)
            frame_array = frame_array.view(1, 3, 64, 64)
            face_data.append(frame_array)
        face_data = torch.cat(face_data, dim=0)
